clj-refactor.el

```diff
@@ -2199,6 +2199,12 @@ the alias in the project."
           (-when-let (long (cljr--aget cljr-magic-require-namespaces short))
             (list short (list long))))))))
 
+(defun cljr--in-keyword-p ()
+  (cljr--is-keyword?
+   (buffer-substring-no-properties
```

Comment:
The argument here is just `cider-symbol-at-point`.

Since we already have `cljr--is-keyword?` perhaps we should make `s` optional and act on the symbol at point otherwise?  I also think we should rename `cljr--is-keyword` ro just `cljr--keyword?` so it makes what we have everywhere else.